### PR TITLE
Creates a Mod List

### PR DESCRIPTION
**Pull request by shedaniel**
Like from https://minecraft.curseforge.com/projects/rift-mod-list but without config.

### CODE REVIEW

File: src/main/java/org/dimdev/rift/mixin/core/client/MixinMainMenu.java [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
 
 @Mixin(GuiMainMenu.class)
-public class MixinMainMenu {
-
-    @Overwrite
-    public void render(int p_render_1_, int p_render_2_, float p_render_3_) {
-        
-    }
+public class MixinMainMenu extends GuiScreen {
+
```
Comment (Runemoro) [inline src/main/java/org/dimdev/rift/mixin/core/client/MixinMainMenu.java]: Please change tabs to spaces

File: src/main/java/org/dimdev/utils/GuiMods.java [hunk 2]
```diff
@@ -1,4 +1,110 @@
 package org.dimdev.utils;
```
Comment (Runemoro) [inline src/main/java/org/dimdev/utils/GuiMods.java]: Move this (and other mod list classes) to `org.dimdev.rift.modlist`

File: src/main/resources/assets/rift/lang/en_us.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "riftmodlist.mods": "Mods List",
```
Comment (Runemoro) [inline src/main/resources/assets/rift/lang/en_us.json]: `riftmodlist` -> `rift.mod_list`

**Comment by Waterpicker** [comment]
Merging in preparation for making the guilist branch prepped for chocohead's 1.13.2 version.

**Pull request merged**